givitsite/gatherer/agora.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+import urllib.request
+import urllib
+from bs4 import BeautifulSoup
+import re
+import csv
+from datetime import datetime
+from friendreq.models import ItemRequest
+
+
+def givit_main():
```

Comment:
i dont understand what you mean :) can you explain? thanks

Reply:
the dicts below - IIRC I saw similar dicts defined in the model classes - and if your definitions will not match you'll get very strange bugs in your app.

Reply:
i try to use it, the problem in that i need dict and in models form its in list of tuples which is not good to me, i need key-value.
**this is how its look in models:**
ITEM_CHOICES = [
    ('20009','closet'),
    ('20016','bed'),
    ('20008', 'chair'),
    ('10006','fridge'),
    ('10029','Washing machine'),
    ('20017','sofa'),
]

when i try to change it to dict its fail with this error- 
default: friendreq.ItemRequest.item: (fields.E005) 'choices' must be an iterable containing (actual value, human readable name) tuples.

actually i prefer to use the "dict" in model it better, so if you have any advise how to do it :)

Reply:
Well you have a couple of choices here:

1. Its very easy to convert a list of pairs like this to a dict on the fly. For example:
 
        dict(ITEM_CHOICES)

    So you could just do that in the beginning of your function.

2. You can use the Django [enumeration types](https://docs.djangoproject.com/en/3.1/ref/models/fields/#enumeration-types) to define the choices, and the you can use `[]` or `()` on the [enum object](https://docs.python.org/3/library/enum.html#module-enum) to do lookups.

Reply:
ok so done,  we decided to chose the first option i use it for one dict, but i still left with 2 dict that translate from english to hebrew because the DB is in english and for the request i need hebrew.